1.0.0-RC.2 Release

BenGalewsky opened the pull request.

mweinberg2718 committed:
Added chunk size info to transformer

Commit:
Improve Awkward Table serialization to Arrow

BenGalewsky committed:
Add docker tag on git tag

gordonwatts committed:
Minor fixes to the readme

gordonwatts committed:
Remove teh update

This was pulling down some bad packages that were breaking the compiler.

gordonwatts committed:
Fix up spelling, formatting

gordonwatts committed:
Change to how we build

Remove the rucioN2N for xcache.

gordonwatts committed:
ATLAS needs to update its contrib rpms

This is an error caused by a bad path in some ATLAS contrib RPM's. Include comments to track the problem.

mweinberg2718 committed:
parent 585fe2af06b9d114861754eaa2d180e92094adb9
author Marc Weinberg <mweinberg2718@gmail.com> 1594910212 -0500
committer Ben Galewsky <ben@peartreestudio.net> 1595255925 -0500

Freeze dependencies for v1.0

gordonwatts commented:
Let's wait to include this. Atlas fixed the bug, and I would like to get that in and finished.

gordonwatts committed:
Do the update before teh build

ATLAS has fixed its bug in the RPM's so we can start doign the update again (see https://sft.its.cern.ch/jira/browse/SPI-1654).

gordonwatts commented:
Ok - with commit https://github.com/ssl-hep/ServiceX_xAOD_CPP_transformer/pull/20/commits/886393d1a2e67310fcb63886812ab2328f0a3dfa, this now good to go from my POV. I've got more updates in flight (like all the C++ `<cmath>` functions), but they do not need to gate RC2.

mweinberg2718 committed:
Change default workdir for bashrc and transform scripts

mweinberg2718 committed:
Changed from "workdir" to more service-specific "servicex"

mweinberg2718 committed:
Change workdir for uproot transformer to match new manager

BenGalewsky committed:
Update status post to new schema

BenGalewsky committed:
Remove references to uproot

gordonwatts reviewed (approved):
Looks great!

BenGalewsky committed:
Reset cwd to /home/atlas to avoid permission problems with /servicex
